src/pages/billing/CreateNewBillingProjectWizard.js

```diff
@@ -134,6 +137,7 @@ const CreateNewBillingProjectWizard = ({ onSuccess, billingAccounts, authorizeAn
               labelStyle: { ...styles.radioButtonLabel },
               onChange: () => {
                 setNextStep(false)
+                Ajax().Metrics.captureEvent(Events.billingProjectWizardStep2, { buttonName: "I don't have access to a Cloud billing account" })
```

Comment:
So this one could be billing:creation:gcpBillingAccountNoAccess` (no buttonName).

Reply:
@admull would you rather see `billing:creation:step2:gcpBillingAccountNoAccess` and `billing:creation:step2:gcpHaveBillingAccount` or the same event names with "step2" left out? (`billing:creation:gcpBillingAccountNoAccess` and `billing:creation:gcpHaveBillingAccount`)?

Reply:
Decision is to keep "step1", "step2", and "step3" as the prefix for those events, but emit it for "step4" events, some of which are fired in general for GCP billing project creation (from the existing dialog, as well as the modal).